libyul/optimiser/OptimizerUtilities.cpp

```diff
@@ -47,3 +49,12 @@ bool yul::isRestrictedIdentifier(Dialect const& _dialect, YulString const& _iden
 {
 	return _identifier.empty() || TokenTraits::isYulKeyword(_identifier.str()) || _dialect.reservedIdentifier(_identifier);
 }
+
+optional<evmasm::Instruction> yul::toEVMInstruction(Dialect const& _dialect, YulString const& _name)
+{
+	if (auto dialect = dynamic_cast<EVMDialect const*>(&_dialect))
+		if (auto builtin = dialect->builtin(_name))
+			if (builtin->instruction)
+				return *builtin->instruction;
```

Comment:
Looks like `instruction` is an `optional` already. No point in unpacking and packing it again:
```suggestion
			return builtin->instruction;
```

Reply:
Good catch!